install/migrations/update_10.0.2_to_10.0.3/inventory.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2022 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+use Glpi\Inventory\Conf;
+
+/**
+ * @var DB $DB
+ * @var Migration $migration
+ */
+
+//new right value for inventory
+//give full rights to profiles having config right
+foreach ($DB->request("glpi_profilerights", "`name` = 'config'") as $profrights) {
+    if ($profrights['rights'] && (Conf::UPDATECONFIG + Conf::IMPORTFROMFILE)) {
+        $rightValue = READ + Conf::UPDATECONFIG + Conf::IMPORTFROMFILE;
+    } else {
+        $rightValue = 0;
+    }
+    $DB->update(
+        "glpi_profilerights",
+        [
+            'rights' => $rightValue
+        ],
+        [
+            'profiles_id' => $profrights['profiles_id'],
+            'name'        => 'inventory'
+        ]
+    );
+}
```

Comment:
I think it will produce the same result.
```suggestion
$migration->addRight('inventory', Conf::UPDATECONFIG);
$migration->addRight('inventory', Conf::IMPORTFROMFILE);
```